supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesConfiguration.scala

```diff
@@ -33,6 +33,6 @@ class JesConfiguration(val configurationDescriptor: BackendConfigurationDescript
   val genomicsFactory = GenomicsFactory(googleConfig.applicationName, jesAuths.genomics, jesAttributes.endpointUrl)
   val dockerCredentials = DockerConfiguration.build(configurationDescriptor.backendConfig).dockerCredentials map JesDockerCredentials.apply
   val needAuthFileUpload = jesAuths.gcs.requiresAuthFile || dockerCredentials.isDefined
-  val qps = jesAttributes.qps
+  val qps: PositiveInt = jesAttributes.qps
```

Comment:
I may or may not sometimes use ascriptions and the subsequent build failures to detect usage...